scripts/extract/parl_json.py

```diff
@@ -1,10 +1,19 @@
 import requests
-import json
 import datetime
-import pandas as pd
+from googleapiclient.http import MediaFileUpload
+import tempfile
+import json
 
 ### Methods ###
 
+def find_folder_id(service, folder_name):
+    query = f"mimeType='application/vnd.google-apps.folder' and name='{folder_name}' and trashed=false"
+    response = service.files().list(q=query, spaces='drive', fields='files(id, name)').execute()
+    files = response.get('files', [])
+    if files:
+        return files[0]['id']  # Assuming the first match is the correct one
```

Comment:
````suggestion
    if files:
        return files[0]['id']  # Assuming the first match is the correct one
    else:
        print(f"Could not find {folder_name}")
````

for logging purposes, to print something back.